### PR TITLE
fix: typos

### PR DESCRIPTION
**Pull request by ASeeto**

### CODE REVIEW

File: website/versioned_docs/version-5.x/server-side-rendering-overview.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ sidebar_label: Overview
...
 
-To accomplish server-side rendering of an SPA, javascript code is executed in NodeJS to generate the initial HTML. In the browser, the same javascript code is executed during a "hydration" process, which attaches event listeners to the HTML. Most popular UI Frameworks (Vue, React, Angular, etc) are capable of executing in both NodeJS and the browser, and offer APIs for both generating the server HTML and hydrating it in the browser. Additionally, there are popular frameworks such as NextJS and Nuxt which simplify the developer experience of server-side rendering.
+To accomplish server-side rendering of a SPA, javascript code is executed in NodeJS to generate the initial HTML. In the browser, the same javascript code is executed during a "hydration" process, which attaches event listeners to the HTML. Most popular UI Frameworks (Vue, React, Angular, etc) are capable of executing in both NodeJS and the browser, and offer APIs for both generating the server HTML and hydrating it in the browser. Additionally, there are popular frameworks such as NextJS and Nuxt which simplify the developer experience of server-side rendering.
```
Comment (frehner) [inline website/versioned_docs/version-5.x/server-side-rendering-overview.md]: I believe `an` is actually correct here, as "SPA" is an acronym and you actually say the letter "ess" out loud. See also https://www.grammar.com/a-vs-an-when-to-use

**Review by frehner — COMMENTED** [review]
Thanks!

File: website/versioned_docs/version-5.x/server-side-rendering-overview.md (hunk 1 above)
Comment (ASeeto) [reply website/versioned_docs/version-5.x/server-side-rendering-overview.md]: Wow haha learn something new every day! Reverted this, thank you

**Review by frehner — APPROVED** [review]

**Comment by frehner** [comment]
Thank you!

File: website/versioned_docs/version-5.x/server-side-rendering-overview.md (hunk 1 above)
Comment (frehner) [reply website/versioned_docs/version-5.x/server-side-rendering-overview.md]: Though maybe enough people say “spa” instead of S.P.A. nowadays… 🤷

**Pull request merged**